src/context/src/lib.rs

```diff
@@ -52,10 +53,10 @@ config!(
 /// An `Arc<Mutex<Context>>` is passed to every `Processor` run by the engine and a `&mut Context` is passed to every `State`
 /// method.
 pub struct Context {
-    // pub video_context: VideoContext,
     pub renderer: Renderer,
     pub asset_manager: AssetManager,
     pub broadcaster: Broadcaster,
+    pub audio_sink: Option<rodio::Sink>,
```

Comment:
Audio implementation can be feature gated, so it can be disabled using the shard system. And I think `rodio::Sink` doesn't need a wrapper unless we want support for multiple audio libraries.

> Also, my limited understanding of rodio gives me that each sink can only play one sound, and there is no way to modify the sound output other than volume.

You can play multiple sounds by mixing sources with `Source::mix` and there are other methods which allow modifying audio output like `Source::fade_in` or `Source::speed` see [docs](http://tomaka.github.io/rodio/rodio/source/trait.Source.html) for `Source` trait. But there is no audio spatialization yet.